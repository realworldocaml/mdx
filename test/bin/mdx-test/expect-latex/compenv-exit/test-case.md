Exits from the toplevel are reported correctly:

```ocaml
# #use "idontexist.ml";;
```
