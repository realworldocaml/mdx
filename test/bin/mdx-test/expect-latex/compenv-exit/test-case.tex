Exits from the toplevel are reported correctly:

\begin{ocaml}
# #use "idontexist.ml";;
\end{ocaml}
