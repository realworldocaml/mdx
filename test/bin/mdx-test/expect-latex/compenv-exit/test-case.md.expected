Exits from the toplevel are reported correctly:

```ocaml
# #use "idontexist.ml";;
Cannot find file idontexist.ml.
[125]
```
